cross/redis/patches/002-lua-env.patch

```diff
@@ -1,5 +1,5 @@
---- deps/lua/src/Makefile.org	2015-10-15 15:44:54.000000000 +0200
-+++ deps/lua/src/Makefile	2015-10-26 10:33:53.716879667 +0100
+--- deps/lua/src/Makefile-orig	2019-12-08 18:34:35.068004000 +0100
```

Comment:
please revert the whole file

Reply:
indeed this part was not reverted. put the original file again, so it should be fine now.